src/sidebar/panel.js

```diff
@@ -28,7 +28,11 @@ enableSync.setAttribute('title', browser.i18n.getMessage('syncNotes'));
 giveFeedbackButton.setAttribute('title', browser.i18n.getMessage('feedback'));
 giveFeedbackMenuItem.text = browser.i18n.getMessage('feedback');
 giveFeedbackButton.setAttribute('href', SURVEY_PATH);
+giveFeedbackButton.setAttribute('target', '_blank');
+giveFeedbackButton.setAttribute('rel', 'noopener');
 giveFeedbackMenuItem.setAttribute('href', SURVEY_PATH);
+giveFeedbackMenuItem.setAttribute('target', '_blank');
```

Comment:
@apoorvaagupta I tried this locally and instead of opening a new tab this new code opens a new window. See the original issue, they expect a new tab instead. Not sure if we can make that happen. 

@Softvision-MariusComan thoughts?

Reply:
Hey!
@vladikoff The value '_blank' of the attribute 'target' opens the link in a new browsing context. It may be a tab(usually) or a new window depending on the browser settings. This setting cant be customised as it is upto the user.

Reply:
The explanation can be viewed under the target section of the following link:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
I think you can use the webextension tabs.open API to make sure to always create a tab.

Reply:
Thankyou @Natim I will just check that and try to work on it. :)